disk/datadog_checks/disk/disk.py

```diff
@@ -264,17 +265,21 @@ def collect_latency_metrics(self):
         for disk_name, disk in iteritems(psutil.disk_io_counters(True)):
             self.log.debug('IO Counters: %s -> %s', disk_name, disk)
             try:
-                # x100 to have it as a percentage,
-                # /1000 as psutil returns the value in ms
-                read_time_pct = disk.read_time * 100 / 1000
-                write_time_pct = disk.write_time * 100 / 1000
                 metric_tags = [] if self._custom_tags is None else self._custom_tags[:]
                 metric_tags.append('device:{}'.format(disk_name))
                 metric_tags.append('device_name:{}'.format(_base_device_name(disk_name)))
                 if self.devices_label.get(disk_name):
                     metric_tags.append(self.devices_label.get(disk_name))
-                self.rate(self.METRIC_DISK.format('read_time_pct'), read_time_pct, tags=metric_tags)
-                self.rate(self.METRIC_DISK.format('write_time_pct'), write_time_pct, tags=metric_tags)
+                if disk_name in self._last_read_time_by_disk:
+                    read_time_since_last_run = disk.read_time - self._last_read_time_by_disk[disk_name]
+                    write_time_since_last_run = disk.write_time - self._last_write_time_by_disk[disk_name]
+                    self.count(self.METRIC_DISK.format('read_time'), read_time_since_last_run, tags=metric_tags)
+                    self.count(self.METRIC_DISK.format('write_time'), write_time_since_last_run, tags=metric_tags)
```

Comment:
Same comment as in the Go version: we could maybe use `self.monotonic_count` here

Reply:
Yes, agree. Any reason not using monotonic_count ?

It seems we will get the same result.